### ISSUE TITLE
notify.alexa_media service has disappeared since last update

### ISSUE THREAD

**Issue by frantzphilippe** [custom-components/alexa_media_player#1878]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/custom-components/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

notify.alexa_media service has disappeared since last update

The last update has suppressed the notify alexa service, many scripts and automatisms have stopped working when they call this notify.alexa_media service.

- Home-assistant (version): last
- alexa_media (version from `const.py` or HA startup): v4.6.2
- alexapy (version from `pip show alexapy` or HA startup): last
- Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). : Yes

**Comment by alandtse** [comment]
Please use  the full template including logs. But most likely you have to reboot. Again without logs, all I can say is leprechauns.

**Comment by dougroutledge** [comment]
Same issue, about 75% of the time after reboot now since the update a few weeks ago.  

Each attempt to use generates this type of error.

Logger: homeassistant.components.automation.front_door_locked
Source: helpers/script.py:409
Integration: Automation ([documentation](https://www.home-assistant.io/integrations/automation), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+automation%22))
First occurred: 2:26:56 PM (1 occurrences)
Last logged: 2:26:56 PM

Front Door Locked: Error executing script. Service not found for call_service at pos 1: Unable to find service notify.alexa_media


While loading it almost appears the service is hanging and HA eventually just moves on.

Logger: homeassistant.bootstrap
Source: bootstrap.py:496
First occurred: 2:25:45 PM (1 occurrences)
Last logged: 2:25:45 PM

Waiting on integrations to complete setup: notify.alexa_media, notify.alexa_media_2



Here is the error I believe that causes it.

This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/alexa_media/notify.py:152
Integration: Alexa Media Player (documentation, issues)
First occurred: 2:26:11 PM (1 occurrences)
Last logged: 2:26:11 PM

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/alexa_media/__init__.py", line 203, in complete_startup
    await notify.async_register_services()
  File "/usr/src/homeassistant/homeassistant/components/notify/legacy.py", line 288, in async_register_services
    if self.targets is not None:
  File "/config/custom_components/alexa_media/notify.py", line 152, in targets
    entity_name = (entity.entity_id).split(".")[1]
AttributeError: 'NoneType' object has no attribute 'split'

**Comment by frantzphilippe** [comment]
Downgrading to 4.6.1 restored the service, waiting for a patch before upgrading, if someone manages to fix it.
Frantz

**Comment by popy2k14** [comment]
thx for the hint.
Also in my case downgrading to 4.6.1 fixed the issue.

**Comment by mspinapolice** [comment]
same issue on 4.6.2.  service appears unavailable and does not show up in the list of servers under developer tools > services.

Confirmed downgrading from 4.6.2 to 4.6.1 restored notify.alexa_media service functionality.

**Comment by level451** [comment]
I've had it happen a second time now, after a bunch of dhcp changes. The
only way I got it back was a reinstall of Alexa media player.

**Comment by popy2k14** [comment]
> Same issue here in multiple Alexa's. Not working since a few days. Reboot doesn't help!
> 
> Could you help me understand how to downgrade to version 4.6.1?

Goto HACS -> search for the integration -> 3 dot menu -> reinstall -> select old version
